docs/versionhistory.rst

```diff
@@ -3,6 +3,11 @@ Version history
 
 This library adheres to `Semantic Versioning 2.0 <https://semver.org/#semantic-versioning-200>`_.
 
+**UNRELEASED**
+
+- Only fill-in empty `body` blocks that may have been left empty by a ``@typechecked`` optimization
+  (`#352 <https://github.com/agronholm/typeguard/issues/352>`_)
+
```

Comment:
If this doesn't fix an existing bug and isn't visible to users, then a changelog entry isn't necessary.
```suggestion
```